libs/@local/graph/store/src/subgraph/temporal_axes.rs

```diff
@@ -256,9 +256,12 @@ impl Default for QueryTemporalAxesUnresolved {
 }
 
 impl QueryTemporalAxesUnresolved {
+    /// Resolves temporal axes relative to the specified timestamp.
+    ///
+    /// Converts unresolved temporal axes into concrete temporal axes by resolving
+    /// relative timestamps and intervals against the provided reference time.
     #[must_use]
-    pub fn resolve(self) -> QueryTemporalAxes {
-        let now = Timestamp::now();
+    pub fn resolve_relative_to(self, now: Timestamp<()>) -> QueryTemporalAxes {
```

Comment:
I don't understand the name. Why do you call it `relative`? Also, I would not call the variable `now` but `timestamp`.
Maybe `fn resolve_with(self, timestamp: Timestamp)`

Reply:
primarily because I couldn't come up with a better name than that. `resolve_with` is a lot better^^

Reply:
fixed in [4603e9e](https://github.com/hashintel/hash/pull/7504/commits/4603e9e4e621e5fd1108b577519a8d044f73d13c)